packages/es-components/README.md

```diff
@@ -31,9 +31,6 @@ import { Component } from 'es-components/lib';
 <head>
   <meta charset="UTF-8">
   <title>My ES Components Site</title>
-  <link rel="stylesheet" href="https://cdn.rawgit.com/WTW-IM/es-assets/8fbaf85d/font.css">
-  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900,900i"
-    rel="stylesheet">
```

Comment:
Before we pull this PR in is there a reason why we want to exclude the font sheets? If the user wants to use the `Icon` component and anything that relies on `Icon` those are important.

Reply:
I forgot about that! WE probably should include them. I'll update.